hathor/manager.py

```diff
@@ -213,6 +217,9 @@ def __init__(self,
         # This is included in some logs to provide more context
         self.environment_info = environment_info
 
+        # Memoize latest best blockchain
+        self._latest_best_blockchain: list[BlockInfo] = []
```

Comment:
It seems we are not using it anymore, right? Maybe delete it.

Reply:
Removed.